### PR TITLE
Update create-build-cluster.sh to make pod utils use workload identit…

### PR DESCRIPTION
**Pull request by cjwagner**
…y and generate default decoration config.

/assign @mpherman2

### CODE REVIEW

File: prow/create-build-cluster.sh [hunk 1]
```diff
@@ -168,26 +191,37 @@ function createUploadSASecret() {
...
   fi
 }
 
-origdir="$( pwd -P )"
-tempdir="$( mktemp -d )"
-echo
-echo "Temporary files produced are stored at: ${tempdir}"
-echo
+function genConfig() {
+  # TODO: Automatically inject this into config.yaml at the same time as kubeconfig credential setup (which auto creates a CL we can include this in).
```
Comment (chaodaiG) [inline prow/create-build-cluster.sh:200]:
```suggestion
  # TODO: Automatically inject this into config.yaml at the same time as kubeconfig credential setup (which auto creates a PR we can include this in).
```

**Review by chaodaiG — APPROVED** [review]
/hold
left a nit

File: prow/create-build-cluster.sh (hunk 1 above)
Comment (cjwagner) [reply prow/create-build-cluster.sh]: Committed.
/label tide/merge-method-squash

**Review by chaodaiG — APPROVED** [review]
/retest

**Pull request merged**